composer.json

```diff
@@ -20,15 +20,16 @@
         }
     ],
     "require": {
-        "php": "~7.1",
+        "php": "~7.2",
         "ext-json": "*",
         "ext-dom": "*",
         "guzzlehttp/guzzle": "^6.3",
         "hamcrest/hamcrest-php": "^2.0",
         "illuminate/support": "^5.5",
         "myclabs/php-enum": "^1.5",
-        "phpunit/phpunit": "^7.1",
-        "symfony/var-dumper": "^4.0"
+        "phpunit/phpunit": "^8.0",
+        "symfony/var-dumper": "^4.0",
+        "dms/phpunit-arraysubset-asserts": "^0.1.0"
```

Comment:
This package contains the assertArraySubset method, independently maintained from PHPUnit. Using it meant the fewest changes to the code.